<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurfaceViewerDemo</class>
 <widget class="QMainWindow" name="SurfaceViewerDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chemkit - Surface Viewer Demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="chemkit::GraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QDockWidget" name="surfaceSettingsDock">
   <property name="windowTitle">
    <string>Surface Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="surfaceTypeLabel">
       <property name="text">
        <string>Surface Type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="surfaceTypeComboBox">
       <item>
        <property name="text">
         <string>Van der Waals</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Solvent Accessible</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="colorModeLabel">
       <property name="text">
        <string>Color Mode:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="opacityLabel">
       <property name="text">
        <string>Opactity:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="opacitySlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="sliderPosition">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="colorModeComboBox">
       <item>
        <property name="text">
         <string>Solid Red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Solid Green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Solid Blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Solid White</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Atom Colored</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Electrostatic Potential</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="probeRadiusLabel">
       <property name="text">
        <string>Probe Radius:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="probeRadiusSpinBox">
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.400000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>chemkit::GraphicsView</class>
   <extends>QWidget</extends>
   <header location="global">chemkit/graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
